src/main/java/Task.java

```diff
@@ -0,0 +1,15 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    // constructor
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+    public String getStatusIcon() {
+
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+    
+}
```

Comment:
all variables and functions are properly named :)

Reply:
Thanks for the feedback :)